### PR TITLE
Replace `:` notation with curly brackets notation `{}` in <RouteHighlighter> element

### LINKED ISSUE #1526: API reference format: Replace `:` notation with curly brackets notation `{}` in `<RouteHighlighter>` element
As part of [formatting our API references](https://github.com/meilisearch/documentation/issues/1351), we will start with replacing `:` notation with curly brackets notation `{}` in the `<RouteHighlighter>` element

Current: 

`/indexes/:index_uid/stats`

New:

`/indexes/{index_uid}/stats`

### PR DESCRIPTION
**Pull request by maryamsulemani97**
closes #1526

### CODE REVIEW

File: learn/advanced/updating.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ Start by using the [get displayed attributes endpoint](/reference/api/displayed_
 ```bash
 # whenever you see :index_uid, replace it with your index's unique id
```
Comment (guimachiavelli) [inline learn/advanced/updating.md:131]: Commenting only here, but all these code blocks have comments that must also be updated.

```suggestion
# whenever you see {index_uid}, replace it with your index's unique id
```

**Review by guimachiavelli — CHANGES_REQUESTED** [review]
Looks good, but we're missing a few `:[parameter/etc]` in code block comments. Once that's fixed, we should be good to go.

File: learn/advanced/updating.md (hunk 1 above)
Comment (maryamsulemani97) [reply learn/advanced/updating.md]: Thanks Gui!

**Review by guimachiavelli — APPROVED** [review]
LGTM 🦖

**Comment by maryamsulemani97** [comment]
bors merge

**Pull request merged**